schema.json

```diff
@@ -0,0 +1,149 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/jesseduffield/lazygit/pkg/config/user-config",
+  "properties": {
+    "defaults": {
+      "additionalProperties": false,
+      "type": "object",
+      "markdownDescription": "Default values used for all presentations. Read more in the [configuration docs](https://github.com/mfontanini/presenterm/blob/master/docs/src/guides/configuration.md)",
+      "properties": {
+        "terminal_font_size": {
+          "type": "number",
+          "minimum": 1,
+          "default": 16,
+          "markdownDescription": "Override the terminal font size when in windows or when using sixel"
+        },
+        "theme": {
+          "anyOf": [
+            {
+              "type": "string",
+              "enum": [
```

Comment:
As an example of what I mean by drifting off, this list can easily get out of sync with the list of supported themes.